xml/System.Collections.Generic/Dictionary`2+AlternateLookup`1.xml

```diff
@@ -32,8 +32,8 @@
     </Attribute>
   </Attributes>
   <Docs>
-    <typeparam name="TAlternateKey">To be added.</typeparam>
-    <summary>To be added.</summary>
+    <typeparam name="TAlternateKey">The alternate type of a key for performing lookups.</typeparam>
+    <summary>Provides a type that may be used to perform operations on a <see cref="T:System.Collections.Generic.Dictionary`2" /> using a <typeparamref name="TAlternateKey" /> as a key instead of a <typeparamref name="TKey" />.</summary>
```

Comment:
```suggestion
    <summary>Provides a type that can be used to perform operations on a <see cref="T:System.Collections.Generic.Dictionary`2" /> using a <typeparamref name="TAlternateKey" /> as a key instead of a <typeparamref name="TKey" />.</summary>
```